cli/README.md

```diff
@@ -35,12 +40,19 @@ Example:
 ```
 
 ### update-cache
-A CLI for the [AMP update-cache](https://developers.google.com/amp/cache/update-cache) API.
 
-It requires the public and private keys to be generated, as [described on the documentation](https://developers.google.com/amp/cache/update-cache#rsa-keys). 
+Uses the [AMP update-cache API](https://developers.google.com/amp/cache/update-cache) to update documents stored in AMP Caches.
 
+It requires the public and private keys to be generated, as [described on the documentation](https://developers.google.com/amp/cache/update-cache#rsa-keys).
```

Comment:
When I read this, I expect the public key also to be a parameter. Maybe explain where the public key needs to be.